rebar.config.sample

```diff
@@ -13,12 +20,6 @@
 %% Erlang compiler options
 {erl_opts, [no_debug_info,
             {i, "myinclude"},
-            %% directories containing source files
-            {src_dirs, ["src", "src2", "src3"]},
```

Comment:
We support it via `{src_dirs, ...}` at the top level of the config, so I decided to aim for that one instead as it's likely better for cross-provider support.